src/app/resources/iis_scripts/admin_api_util.ps1

```diff
@@ -0,0 +1,126 @@
+        Add-LocalGroupMember -Group $group -Member $userOrGroup | Out-Null
+    }
+}
+
+####################### Main script ################################
+
+## TODO: Consider creating a lock around this script because it is not concurrency safe
+
+if ($install) {
+    InstallAPI
+}
+
+$service = Get-WmiObject win32_service | Where-Object {$_.name -eq $serviceName}
+if ($service.StartInfo.EnvironmentVariables -and $service.StartInfo.EnvironmentVariables["USE_CURRENT_DIRECTORY_AS_ROOT"] -and $service.StartInfo.WorkingDirectory) {
+    $workingDirectory = $service.StartInfo.WorkingDirectory
+} else {
+    $proc = Get-Process -id $service.ProcessId
+    $workingDirectory = Split-Path ($proc.Modules | Where-Object { $_.ModuleName -eq "Microsoft.IIS.Administration.dll" }).FileName
+}
+
+$configLocation = [System.IO.Path]::Combine($workingDirectory, "config", "appsettings.json")
+$config = Get-Content -Raw -Path $configLocation | ConvertFrom-Json
+
+$user = $(whoami)
+EnsureGroup $iisAdminOwners
+EnsureMember $iisAdminOwners $user
+
+if (!$config.security.users.owners.Contains($iisAdminOwners)) {
+    foreach ($member in $config.security.users.owners) {
+        EnsureMember $iisAdminOwners $member
+    }
+    $config.security.users.owners = @( $iisAdminOwners )
+    $updateConfig = $true
+}
+
+if ($updateConfig) {
+    $apiHome = [System.IO.Path]::Combine($workingDirectory, "..")
+    ## Installer added a read-only rule on current user to the directory, delete it
+    $dirAcl = Get-Acl $apiHome
+    foreach ($access in $dirAcl.Access | Where-Object { $_.IdentityReference.Value -eq $user }) {
+        $dirAcl.RemoveAccessRule($access) | Out-Null
+    }
+    Set-Acl -Path $apiHome -AclObject $dirAcl | Out-Null
+
+    $rights = [System.Security.AccessControl.FileSystemRights]::Traverse `
+        -bOr [System.Security.AccessControl.FileSystemRights]::Modify `
+        -bOr [System.Security.AccessControl.FileSystemRights]::ChangePermissions
+
+    $dirAccessGranted = $dirAcl.Access | Where-Object {(($_.IdentityReference.Value -eq "${env:ComputerName}\${iisAdminOwners}") -and (($_.FileSystemRights -bAnd $rights) -eq $rights))}
+    if (!$dirAccessGranted) {
+        $inheritFlags = [System.Security.AccessControl.InheritanceFlags]::ContainerInherit `
+            -bOr [System.Security.AccessControl.InheritanceFlags]::ObjectInherit
+        $propFlags = [System.Security.AccessControl.PropagationFlags]::InheritOnly
+        $ar = New-Object System.Security.AccessControl.FileSystemAccessRule($iisAdminOwners, $rights, $inheritFlags, $propFlags, "allow")
+        $dirAcl.SetAccessRule($ar) | Out-Null
+        $ar = New-Object System.Security.AccessControl.FileSystemAccessRule("BUILTIN\Administrators", $rights, $inheritFlags, $propFlags, "allow")
+        $dirAcl.SetAccessRule($ar) | Out-Null
+        $dirAcl.SetOwner([System.Security.Principal.NTAccount]$iisAdminOwners) | Out-Null

```

Comment:
Setting the owner of the IIS Administration API files to IIS Administration Owners? I wouldn't think we would want to go any less than administrators..